build-tools/automation/build.groovy

```diff
@@ -131,11 +131,15 @@ timestamps {
             }
         }
 
-        stageWithTimeout('publish test error logs to Azure', 10, 'MINUTES', XADir, false) {  // Typically takes less than a minute
+        stageWithTimeout('publish test error logs to Azure', 10, 'MINUTES', '', false) {  // Typically takes less than a minute
             echo "packaging test error logs"
-            sh "make package-test-errors"
 
-            def publishTestFilePaths = "xa-test-errors*"
+            sh """
+                cd ${XADir}
+                make package-test-errors
```

Comment:
could this also be `sh "make -C ${XADir} -k package-test-errors"`?

Reply:
indeed that should work. testing the change now. commit https://github.com/xamarin/xamarin-android/pull/2766/commits/d85f005712532a916803db9418cae7f1cf08f7dd

Reply:
build step executed as expected with test build `https://jenkins.mono-project.com/view/Xamarin.Android/job/xamarin-android-pipe-eval/9/`